extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/actionTypeExtractors.scala

```diff
@@ -40,3 +41,13 @@ class OverwriteOrInsertActionTypeExtractor extends ActionTypeExtractor {
     if (v1.asInstanceOf[Boolean]) INSERT_OVERWRITE else INSERT
   }
 }
+
+class SaveModeActionTypeExtractor extends ActionTypeExtractor {
+  override def apply(v1: AnyRef): PrivilegeObjectActionType = {
+    invokeAs[String](v1, "name") match {
```

Comment:
SaveMode is a stable API for end users, it's to match directly.

Reply:
Do you mean import SaveMode directly and call the name method of it?

Reply:
Addressed.

Reply:
OK. Import `SaveMode` and use the name of `SaveMode` Enum for matching, while unable to match directly to Java enum in Scala.